nanome/_internal/_structure/_atom.py

```diff
@@ -25,6 +25,7 @@ def __init__(self):
         self._bfactor = 0.0
         self._acceptor = False
         self._donor = False
+        self._polar_hydrogen = False
```

Comment:
I know we're really trying to release and so this makes sense in our current situation, but I think we might be able to do something clever later and remove this: I wanna revisit the polar hydrogens feature with partial charge and plugin-defined macros (plugin_instance.macro. ...) that can be registered to a core client like plugin integrations are currently so that they can be run by UI components. I also think it would be great for menus, layout_nodes, and content to make quick state transitions as specified by a plugin (but not executed by it). Can you, me and @shessenauer talk about this at some point in the next maybe 3 months?